docs/DatabaseCatalogAPI.yaml

```diff
@@ -160,6 +160,26 @@ paths:
               schema:
                 $ref: "#/components/schemas/BasicDatasets"
 
+  /datasets/{id}:
+    parameters:
+      - $ref: "#/components/parameters/datasetIdPathParam"
+    get:
+      description: Get the specified dataset from the Mobility Database.
+      tags:
+        - "datasets"
+
+      security:
+        - ApiKeyAuth: []
+      responses:
+        200:
+          description: >
+            Successful pull of the dataset common info for the provided ID.
+            This info has a reduced set of fields that are common to all types of dataset.
```

Comment:
Ideally this includes all fields specific to the data type. If it's a reduced set of fields, I don't think it has much value to include, and it makes more sense to just use the data type specific end point, e.g `/datasets/gtfs/{id}` Thoughts? Having a reduced set of fields for `/datasets` and `/feeds` makes sense to me since the response will be huge, but not for id specific requests. cc @jcpitre @davidgamez

Reply:
I  believe the rest api best practices suggests that the data returned from a given endpoint should always use the same schema. It's not an obligation but I think we would need a better reason to stray from that.
One case is if for some reason you have the ID of a feed or dataset, but nothing else. You will then be able to get the reduced feed/dataset that will give you the type, then you can get the full data. A bit convoluted maybe, but it's a corner case IMO. 
Maybe in the spirit of an MVP we could get rid of /feeds/{id} and /datasets/{id} altogether. If the case described presents itself, the workaround would be to get the whole list then keep only the one with the requested ID.

Reply:
Got it! I understand now the need for a feed type specific end point given the rest API best practices and the processing required. Thanks for educating me @jcpitre @davidgamez on that. :smile: